### PR TITLE
add antivirus

### PR DESCRIPTION
**Pull request by marcom4rtinez**
@sevigrimm some inputs? i ran out of ideas for this article

### CODE REVIEW

File: best-practices/content/30_Antivirus.tex [hunk 1]
```diff
@@ -0,0 +1,64 @@
+\chapter{\acrlong{av}}
+\section{Allgemeines} %TODO: maybe change title
+Ein \acrfull{av} ist der erste Verteidigungsmechanismus in einem Netzwerk. Angriffe, darunter Schadsoftware, kommen in den meinsten Fällen über einen Endpoint (z.B. Notebook) in ein Firmennetzwerk.\\
+
+Eine gegebene Struktur ist immer nur so stabil, wie sein schwächstes Glied. Dies gilt bei Häusern wie auch bei Computernetzwerken.
+Oftmals ist das schwächste Glied ein Client oder ein Server, da diese Geräte vielfach mit dem Internet kommunizieren.
+Dies macht genau diese Geräte besonders schützenswert.\\
+
+Server \& Clients werden mit einem \acrlong{av} versucht zu schützen. Ein Antivirus hilft bekannte Schadsoftware zu entfernen.
```
Comment (sevigrimm) [inline best-practices/content/30_Antivirus.tex:9]:
```suggestion
Server \& Clients werden mit einem \acrlong{av} versucht zu schützen. Ein Antivirus hilft bekannte Schadsoftware zu detektieren und entfernen.
```

File: best-practices/content/30_Antivirus.tex [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+
+
+\section{Windows Defender}
+\acrfull{av} kommen in allen Arten \& Formen.
+Es ist schwer den Überblick über die Landschaft der Antiviren Programme zu behalten.\\
+
+
+Microsoft bietet für seine Produkte einen eigenen kostenfreien \acrlong{av} an.
+Der Windows Defender ist weder der beste \acrshort{av} noch hat er die meisten Features.
```
Comment (sevigrimm) [inline best-practices/content/30_Antivirus.tex:25]: Würi jetzt so ned sege:

https://www.microsoft.com/security/blog/2021/05/11/gartner-names-microsoft-a-leader-in-the-2021-endpoint-protection-platforms-magic-quadrant/

File: best-practices/content/30_Antivirus.tex [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        \end{table}
+    \end{multicols}
+\end{minipage}
+%TODO: @sevigrimm add more points?
+
+
+
+
+\subsection{Wazuh Integration}
+%TODO: @sevigrimm already included correct?
```
Comment (sevigrimm) [inline best-practices/content/30_Antivirus.tex:64]: ich schrieb d Integration no

File: best-practices/content/30_Antivirus.tex (hunk 2 above)
Comment (marcom4rtinez) [reply best-practices/content/30_Antivirus.tex]: review again @sevigrimm

**Pull request merged**